packages/library/src/lib/plugins/backend.ts

```diff
@@ -439,7 +439,7 @@ export function mergeHTMLFragment(
 
         const revisedHTML = modifiedTarget.outerHTML
 
-        if (originalHTML !== revisedHTML) {
+        if (originalHTML !== revisedHTML && settleTime > 0) {
```

Comment:
@delaneyj I added a condition to only add/remove the settling class if the settle time is non-zero. Please review if you’re ok with this.